app/views/katello/api/v2/capsule_content/sync_status.json.rabl

```diff
@@ -18,15 +18,13 @@ child @lifecycle_environments => :lifecycle_environments do
     @capsule.environment_syncable?(env)
   end
 
-  if @capsule.has_feature?(SmartProxy::PULP_NODE_FEATURE)
+  if @capsule.has_feature?(SmartProxy::PULP_NODE_FEATURE) || @capsule.has_feature?(SmartProxy::PULPCORE)
```

Comment:
this should be PULP3_FEATURE  (sorry i typed it from memory in my comment and got it wrong)